docs/community/mxnet_channels.md

```diff
@@ -0,0 +1,6 @@
+# Join MXNet Development Discussion
+
+You can join the MXNet development discussion
+
+- On the [MXNet Apache mailing list](https://lists.apache.org/list.html?dev@mxnet.apache.org): `dev@mxnet.apache.org`. To subscribe, send an email to `dev-subscribe@mxnet.apache.org`.
+- On the [MXNet Slack channel](https://apache-mxnet.slack.com): To request an invitation to the channel please email: `dev@mxnet.apache.org`.
```

Comment:
It's not good to have all the channels start with "On the ..."

Better to change the prompt above to 
"Converse with the MXNet community via the following channels: "
- [MXNet Apache mailing list](https://lists.apache.org/list.html?dev@mxnet.apache.org)  (dev@mxnet.apache.org). To subscribe, send an email to dev-subscribe@mxnet.apache.org.
- [MXNet Slack channel](https://apache-mxnet.slack.com): To request an invitation to the channel please email: 
 dev@mxnet.apache.org.

Note those email addresses need to be mailto links (I believe Markdown (on GitHub at least) converts automatically), not code snippets.